packages/fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -781,9 +783,15 @@ export class ForkChoice implements IForkChoice {
     let block = this.getBlock(headBlockHash);
     if (!block) return null;
     const {slot} = block;
+    const blockEpoch = computeEpochAtSlot(slot);
+
     // The shuffling is determined by the block at the end of the target epoch
     // minus the shuffling lookahead (usually 2). We call this the "pivot".
-    const pivotSlot = computeStartSlotAtEpoch(computeEpochAtSlot(slot) - 1) - 1;
+    const pivotSlot = computeEndSlotAtEpoch(blockEpoch - MIN_SEED_LOOKAHEAD);
```

Comment:
As the description suggests we need to look head from the end of target epoch ( minus seed look ahead). Earlier expression was subtracting to two epoch.

Reply:
@nazarhussain  the code already goes with the description, if you want to use the new util, it should be

```typescript
const pivotSlot = computeEndSlotAtEpoch(blockEpoch - MIN_SEED_LOOKAHEAD - 1);
```

@dapplion could also help double check this

Reply:
I think, we need to start looking past in epoch with minimum seed look ahead value `MIN_SEED_LOOKAHEAD`. If we want to skip past one extra epoch we should change `MIN_SEED_LOOKAHEAD` param.